### DISCUSSION TITLE
Normalization of data from replicates

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#1805] [General]
@rcorces 
Hi Ryan,

When running getMarkerFeatures(proj) on replicates for a given treatment I obtain big differences when querying for specific markers using plotMarkerHeatmap(). This is confounding the inference from different treatments.

se <- getMarkerFeatures(ArchRProj = proj, groupBy = "Sample", useMatrix = "GeneExpressionMatrix", ### bias =c("TSSEnrichment", "log10(Gex_nUMI)"))

Outside of bias adjustments what can be done to address this issue?  Can you help with some suggestions.

Thank you!

**Comment by rcorces** [comment]
I'm not sure I understand your issue. Perhaps you can read this previous post and clarify.
https://github.com/GreenleafLab/ArchR/pull/1409

**Comment by sylestiel** [comment]
@rcorces

Despite running
proj <- addHarmony(
ArchRProj = proj,
reducedDims ="LSI_Combined",
name = "Harmony",
groupBy = "Sample"
)

when i look to plot specific markers using the Gene Expression or Gene Score Matrix I see a lot of sample to sample variation. As seen here
<img width="580" alt="Screen Shot 2023-01-17 at 11 14 26 AM" src="https://user-images.githubusercontent.com/64283689/212979729-90684ec1-c920-4e03-b35f-ad4dd2277a80.png">

**Reply by rcorces** [reply]
is your implication that these values are incorrect?

**Reply by sylestiel** [reply]
Well, I assumed that by running Harmony the replicate samples would show more of a conformity in expression whether we query for GeneScores or GeneExpression.
 Is there some way to tweak this function besides using the bias =c("TSSEnrichment", "log10(Gex_nUMI)")) or bias =c("TSSEnrichment", "log10(nFrags)")) parameters?

Your thoughts/insights/suggestions would be much appreciated.

Thanks!

**Reply by rcorces** [reply]
Harmony does not alter any of the underlying data, just the positioning within the dimensionality reduction. I dont expect that adjusting any arguments will solve this.

**Reply by sylestiel** [reply]
Okay. Thanks!
